backend/commons/src/test/java/org/sonarsource/sonarlint/core/commons/GitUtilsTest.java

```diff
@@ -62,6 +71,24 @@ class GitUtilsTest {
   @TempDir
   private Path projectDirPath;
   private Git git;
+  private static Path bareRepoPath;
+  private static Path workingRepoPath;
+
+  @BeforeAll
+  public static void beforeAll() throws GitAPIException, IOException {
+    setUpBareRepo(Map.of(".gitignore", "*.log\n*.tmp\n"));
+  }
+
+  @AfterAll
+  @DisabledOnOs(OS.WINDOWS)
+  public static void afterAll() {
+    try {
+      FileUtils.forceDelete(bareRepoPath.toFile());
+      FileUtils.forceDelete(workingRepoPath.toFile());
+    } catch (Exception ignored) {
+      //It throws an exception in windows
```

Comment:
If it's disabled, you can just add an exception to the method signature.

Reply:
Ah, yes, that was my plan. I forgot to remove it. Thanks for the remainder.

Reply:
Didn't work that way. I remove the annotation for disabling windows. Now the tests are green